node/api/daemon.go

```diff
@@ -2,5 +2,7 @@ package api
 
 // DaemonVersionGet contains information about the running daemon's version.
 type DaemonVersionGet struct {
-	Version string
+	Version     string
+	GitRevision string
+	BuildTime   string
```

Comment:
these will need to be documented, but we can handle that in a separate PR. I want to include this change in the rc